### DISCUSSION TITLE
a function to add elements from two lists one by one

### DISCUSSION THREAD

**Discussion by l1t1** [duckdb/duckdb#5754] [Ideas]
let a=[1,2], b=[3,4]
I need the result of[1+3,2+4]
in python
```
>>> a=[1,2]
>>> b=[3,4]
>>> import numpy as np
>>> result = np.sum([a, b], axis=0).tolist()
>>> print(result)
[4, 6]
```

in duckdb, I did so, and the result isn't as expected.
```
select [i[1]+i[2] for i in [[1, 2], [3, 4]]];
 [3, 7]
```

**Comment by l1t1** [comment]
this works, but isnt simple
```
select list(k) from(select unnest([1, 2])i,unnest([3,4])j, i+j k);
[4, 6]
```

**Comment by l1t1** [comment]
in python, zip function works
```
>>> [i+j for i,j in zip(a,b)]
[4, 6]
```

**Comment by taniabogatsch** [comment]
Hi! 
```sql
select [i[1]+i[2] for i in [[1, 2], [3, 4]]];
```
This query reads like 'for each list in the list of lists, do...'. So you are summing up the first two elements of each list, but no cross-list summation happens.

We do not yet support zipping lists, it is on the maybe roadmap (https://github.com/duckdb/duckdb/issues/1940). Until then, I am afraid that you have to use a workaround using e.g. `UNNEST`, as you did.
